src/block/utils/Fallable.php

```diff
@@ -33,4 +34,9 @@ interface Fallable{
 	 * Return null if you don't want to change the usual behaviour.
 	 */
 	public function tickFalling() : ?Block;
+
+	/**
+	 * Returns the sound that will be played when FallingBlock hits the ground.
+	 */
+	public function getFallSound() : ?Sound;
```

Comment:
This is BC-breaking and must be targeted at `next-major`.

Reply:
There is a default implementation on FallableTrait: https://github.com/IvanCraft623/PocketMine-MP/blob/fallingblock-sounds/src/block/utils/FallableTrait.php#L63, so existing code should still work.

Reply:
You can't assume that implementations will use this trait.

Reply:
done.

Reply:
I think `getLandSound()` would be more clear. Coming back to this just now, I briefly thought that "fall sound" meant "sound made while falling".

Reply:
> I think `getLandSound()` would be more clear. Coming back to this just now, I briefly thought that "fall sound" meant "sound made while falling".

done